src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -6495,7 +6495,8 @@ public static void Main()
             CleanupAllGeneratedFiles(rsp);
         }
 
-        [Fact, WorkItem(530024, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/530024")]
+        // PROTOTYPE(NullableReferenceTypes): Do not emit [module: NullablAttribute] if no nullable types.
```

Comment:
As we discussed, I think the `NullableAttribute` should always be emitted on the assembly when the "nullable" feature is turned on.

Reply:
Perhaps, although we'd want emit to succeed in the case where `System.Attribute` is not available.

---
In reply to: [171079718](https://github.com/dotnet/roslyn/pull/25087#discussion_r171079718) [](ancestors = 171079718)

Reply:
>Perhaps, although we'd want emit to succeed in the case where System.Attribute is not available.

Perhaps that should be a warning, if the only place where we wanted the nullable attribute is the module..

---
In reply to: [171104358](https://github.com/dotnet/roslyn/pull/25087#discussion_r171104358) [](ancestors = 171104358,171079718)

Reply:
Nit: I agree with the discussion thread, but it seems to me the comment doesn't capture the desired behavior which we're discussing. There should be no "if no nullable types" case. 

---
In reply to: [171140137](https://github.com/dotnet/roslyn/pull/25087#discussion_r171140137) [](ancestors = 171140137,171139914,171104358,171079718)